courses/machine_learning/deepdive2/structured/solutions/4c_keras_wide_and_deep_babyweight.ipynb

```diff
@@ -0,0 +1,814 @@
+{
```

Comment:
<p>notice we're hard coding the boundary values for "mother_age" and "gestation_weeks". This was how it was done in the previous iteration of this lab too.</p><p> </p><p>I guess in this situation it makes sense since it pertains to the specific features and preferrable to using a global variable that can be set outside the "create_feature_columns" function. Not sure the best practice in this scenario.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I intended students to choose this lab or the DNN version. Also, I didn't want to confuse things by bringing in a bunch of extra hyperparameters like min_mother_age, max_mother_age, min_gest_weeks, max_gest_weeks since the main point of the lab is to just get more used to making wide and deep feature columns and not productionizing everything to be passed parameters.</p>